device/bgp/bgpsrtev4remoteendpointsubtlv.yaml

```diff
@@ -3,6 +3,9 @@ components:
     BgpSrte.RemoteEndpointSubTlv:
       description: >-
         Configuration for BGP remote endpoint sub TLV settings.
+        Question: Do we need to keep? Draft says 
```

Comment:
Anything that is not must is probably better removed. Add it only if someone asks for negative use-case testability.

Reply:
ok. noted your comment.